src/test/data/JsonSerializableAddressBookTest/typicalPersonsAddressBook.json

```diff
@@ -5,42 +5,49 @@
     "phone" : "94351253",
     "email" : "alice@example.com",
     "address" : "123, Jurong West Ave 6, #08-111",
-    "tagged" : [ "friends" ]
+    "tagged" : [ "friends" ],
+    "deliveryDate": "01/01/2022"
```

Comment:
Perhaps you could consider using different dates here so there's more variety in the tests?

Reply:
Yup definitely! If I recall correctly Benson's details will be used specifically in some tests. So since you are adding a field as well, take note of those specific test cases that tries to extract Benson's details ya. Probably don't have to do anything about it if you didn't miss anything ~

Reply:
Thanks for the heads up :)